src/convert.jl

```diff
@@ -202,6 +202,26 @@ function trixi2vtk(filename::AbstractString...;
             verbose && println("| | Element variable: $label...")
             @timeit label vtk_celldata[label] = variable
           end
+
+          # Add node variables
+          for (label, variable) in node_variables
+            verbose && println("| | Node variable: $label...")
+            if reinterpolate
+              if index == 1
+                println("WARNING: The limiting coefficients are no continuous field but happens " *
+                "to be represented by a piecewise-constant approximation. Thus, reinterpolation " *
+                "does not give a meaningful representation.")
+              end
+              @timeit "interpolate data" interpolated_cell_data = interpolate_data(Val(format),
+                                                                    reshape(variable, size(variable)..., 1),
+                                                                    mesh, n_visnodes, verbose)
+            else
+              @timeit "interpolate data" interpolated_cell_data = reshape(variable,
+                                                                          n_visnodes^ndims_ * n_elements)
+            end
+            # Add the "interpolated" cell_data to celldata, not node_data
+            @timeit label vtk_nodedata[label] = interpolated_cell_data
```

Comment:
```suggestion
            # Add to node_data
            @timeit label vtk_nodedata[label] = interpolated_cell_data
```
?

Reply:
Okay, that doesn't make any sense. Thanks

Reply:
Done in [526b7ce](https://github.com/trixi-framework/Trixi2Vtk.jl/pull/67/commits/526b7cec93696b3416d83e173da4b27a0e8a91b1)